Source/Core/DolphinWX/ConfigMain.cpp

```diff
@@ -297,32 +299,34 @@ void CConfigMain::InitializeGUILists()
 
 	// GUI language arrayStrings
 	// keep these in sync with the langIds array at the beginning of this file
-	arrayStringFor_InterfaceLang.Add(_("<System>"));
-	arrayStringFor_InterfaceLang.Add(_("Arabic"));
-	arrayStringFor_InterfaceLang.Add(_("Catalan"));
-	arrayStringFor_InterfaceLang.Add(_("Chinese (Simplified)"));
-	arrayStringFor_InterfaceLang.Add(_("Chinese (Traditional)"));
-	arrayStringFor_InterfaceLang.Add(_("Czech"));
-	arrayStringFor_InterfaceLang.Add(_("Dutch"));
-	arrayStringFor_InterfaceLang.Add(_("English"));
-	arrayStringFor_InterfaceLang.Add(_("Farsi"));
-	arrayStringFor_InterfaceLang.Add(_("French"));
-	arrayStringFor_InterfaceLang.Add(_("German"));
-	arrayStringFor_InterfaceLang.Add(_("Greek"));
-	arrayStringFor_InterfaceLang.Add(_("Hebrew"));
-	arrayStringFor_InterfaceLang.Add(_("Hungarian"));
-	arrayStringFor_InterfaceLang.Add(_("Italian"));
-	arrayStringFor_InterfaceLang.Add(_("Japanese"));
-	arrayStringFor_InterfaceLang.Add(_("Korean"));
-	arrayStringFor_InterfaceLang.Add(_("Norwegian Bokmaal"));
-	arrayStringFor_InterfaceLang.Add(_("Polish"));
-	arrayStringFor_InterfaceLang.Add(_("Portuguese"));
-	arrayStringFor_InterfaceLang.Add(_("Portuguese (Brazilian)"));
-	arrayStringFor_InterfaceLang.Add(_("Russian"));
-	arrayStringFor_InterfaceLang.Add(_("Serbian"));
-	arrayStringFor_InterfaceLang.Add(_("Spanish"));
-	arrayStringFor_InterfaceLang.Add(_("Swedish"));
-	arrayStringFor_InterfaceLang.Add(_("Turkish"));
+	arrayStringFor_InterfaceLang.Add(_("<System Language>"));
+
+	arrayStringFor_InterfaceLang.Add(L"Catal\u00E0");                                       // Catalan
```

Comment:
The L is necessary on Windows (or any other non-UTF8 system): otherwise, it won't encode the string literal correctly.  It has nothing to do with the internal representation of a wxString.